cron_tasks_folder/daily8am/send-mail

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# This test email is temporary to be replaced with the result of:
+# https://github.com/elifesciences/elife-xpub/blob/develop/scripts/prod-status.sh
```

Comment:
Keeping in mind portability between Xpub and Libero Reviewer fromhttps://github.com/libero/reviewer-reporter/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R6, then https://github.com/elifesciences/elife-xpub/blob/develop/scripts/prod-status.sh#L37-L38 has two problems:
- authenticates via SSH with another node (actually its host), but this container doesn't have a private key to use for that (probably a `ssh` binary is there or can be added easily)
- assumes there is a node to SSH in, which isn't true for Libero Reviewer as it runs on Kubernetes

Two possible solutions:
1. this repository is moved into `elifesciences/` as an eLife-specific solution that will need update to switch to Libero Reviewer
2. the SQL queries are executed directly inside this container with a `psql` client, the necessary `PG*` credentials being passed in

2 still leaves vulnerable to schema changes, but the effect is that this script breaks so lower risk that causing downtimes or failed deployments.

Reply:
Sorry you've not been involved in all the discussion around this.

I didn't want to write too much in the comment - but currently its designed to run from my laptop hence the carpaccio approach.  This script has been running since the May 2019 so we are aware of the changes it needs to have to be schema compatible. Thanks

Reply:
I can only assume path 2 will be the one taken, otherwise the Github organizaton would have been different from the start.

Reply:
I've not really thought that deeply about this just yet - but yes hoping the psql client can go into the container and use the config from deployment to connect